src/SFA.DAS.EmployerApprenticeshipsService.Domain/Interfaces/IFeatureService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EAS.Domain.Models.FeatureToggles;
+
+namespace SFA.DAS.EAS.Domain.Interfaces
+{
+    /// <summary>
+    ///     Provides access to feature and associated controller actions.
+    /// </summary>
+    public interface IFeatureService
```

Comment:
Could this interface expose just the `GetFeatureThatAllowsAccessToOperationAsync` method instead and allow the `FeatureAgreementService` & `FeatureWhiteListingService` to make the decisions?

Reply:
Done